content/docs/hooks-state.md

```diff
@@ -28,11 +28,11 @@ function Example() {
 }
 ```
 
-We'll start learning about Hooks by comparing this code to an equivalent class example.
+Vamos começar aprendendo sobre Hooks comparando este código com um exemplo equivalente a classe.
```

Comment:
```suggestion
Vamos começar aprendendo sobre Hooks comparando este código com um exemplo de classe equivalente.
```

Reply:
Acho que seria melhor alterar para => `Vamos começar aprendendo sobre Hooks comparando este código com um exemplo equivalente utilizando classe.`

Reply:
Achei `de classe equivalente` um pouco confuso.

@Lukyhenson gostei da sugestão.

Reply:
Também achei confuso, acho que com o `utilizando` fica mais claro o q queremos fazer

Reply:
Talvez fique legal também: `Vamos começar aprendendo sobre Hooks comparando este código com um exemplo equivalente a um componente de classe.`

Reply:
acho q ter a palavra componente fica meio redundante nesse contexto, pq quem está tentando aprender hooks teoricamente ja sabe como funciona os functional and class components (deveria saber kkk), e q o hooks está relacionado com um functional component equivalente a um de classe. na minha opinião claro.

Reply:
É, acredito que para quem já esteja acostumado com functional e class components entenda bem sem problemas, mas acho que fica um pouco mais claro pra quem não conhece a diferença da estrutura de um ou outro.

Reply:
@abnersajr

Reply:
```suggestion
Vamos começar a aprender sobre Hooks comparando este código com um exemplo equivalente utilizando Class.
```

ou

```suggestion
Vamos começar a aprender sobre Hooks comparando este código com um exemplo equivalente utilizando classe.
```

Reply:
Sobre o ponto do `aprendendo` essa seria a tradução literal de `learning`. No entanto fica melhor da forma sugerida acima.

Reply:
acho q  nao teria problema usar classe ja que deixa implicito o termo class do javascript, na minha opinião a segunda opção faz mais sentido pra mim :smile:

Reply:
O termo `classe` é uma tradução recomendada [aqui](https://github.com/reactjs/pt-BR.reactjs.org/blob/master/GLOSSARY.md) e acho que já está bem padronizado, sobre o `learning` tem total razão.

Reply:
então assim fica melhor mesmo.
```suggestion
Vamos começar a aprender sobre Hooks comparando este código com um exemplo equivalente utilizando classe.
```